smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/SwiftWriter.kt

```diff
@@ -90,6 +90,7 @@ class SwiftWriter(private val fullPackageName: String, swiftImportContainer: Swi
                         sectionWriter.write(codeWriter, previousValue)
                     }
                 }
+                GENERATED_FILE_HEADER = integration.copyrightNotice() ?: GENERATED_FILE_HEADER
```

Comment:
I think we want to keep the `Code generated by smithy-swift-codegen` notice, but also add in the custom header right?

Reply:
Yeah, but that's done not by appending but just including it in AWS side as well.

Reply:
I guess it allows customers to use their own "do not edit" warning as well, if they want to.